src/engine/strat_engine/cleanup.rs

```diff
@@ -14,11 +13,11 @@ use super::pool::StratPool;
 /// Wipe a Vec of blockdevs of their identifying headers.
 /// Return an error if any of the blockdevs could not be wiped.
 /// If an error occurs while wiping a blockdev, attempt to wipe all remaining.
-pub fn wipe_blockdevs(mut blockdevs: Vec<BlockDev>) -> EngineResult<()> {
+pub fn wipe_blockdevs(blockdevs: &[BlockDev]) -> EngineResult<()> {
```

Comment:
The header comment will need updating...

Reply:
ok